samples/excel/worksheet/hide-unhide-worksheet.yaml

```diff
@@ -0,0 +1,168 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Hide and unhide worksheet
+description: Explore the APIs to hide and unhide a worksheet
+script:
+  content: >-
+    $("#hide-worksheet").click(hideWorksheet);
+
+    $("#unhide-worksheet").click(unhideWorksheet);
+
+
+    async function hideWorksheet() {
+        try {
+            await Excel.run(async (context) => {
+
+                let visibleSheets = await filterWorksheetsByVisibility(context, Excel.SheetVisibility.visible);
+
+                if (visibleSheets.length > 1) {
+                    console.log(`Hiding worksheet named "${visibleSheets[0].name}"...`);
+
+                    visibleSheets[0].visibility = Excel.SheetVisibility.hidden;
+
+                    await context.sync();
+
+                } else {
+                    console.log("Unable to hide the last visible worksheet in the workbook");
+                }
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+
+    async function unhideWorksheet() {
+        try {
+            await Excel.run(async (context) => {
+
+                let visibleSheets = await filterWorksheetsByVisibility(context, Excel.SheetVisibility.hidden);
+
+                if (visibleSheets.length > 0) {
+                    console.log(`Unhiding worksheet named "${visibleSheets[0].name}"...`);
+
+                    visibleSheets[0].visibility = Excel.SheetVisibility.visible;
+
+                    await context.sync();
+                    
+                } else {
+                    console.log("No hidden worksheets in the workbook");
+                }
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+
+    async function filterWorksheetsByVisibility(context: Excel.RequestContext,
+    visibility: string): Promise<Excel.Worksheet[]> {
+
+        let sheets = context.workbook.worksheets;
+        sheets.load("items/name, items/visibility");
```

Comment:
FYI, The "canonical" form doesn't actually need "items/" (and RTM didn't have it, though it should get its JS updated before we release).  I personally like "items/" prefix, because I think it's easier to explain, but just figured I'd let you know.